features/shop/product/searching/filtering_list_of_products_by_options.feature

```diff
@@ -0,0 +1,28 @@
+@searching_products
+Feature: Filtering list of products by options
+    In order to see more specific list of the products
+    As an Visitor
+    I want to be able to filter the products
+
+    Background:
+        Given the store has about 10 Mugs, 20 Stickers and 25 Books
+
+    @domain
+    Scenario: Filtering products by their type
+        When I filter them by double mug type
+        Then I should see 10 products on the list
+
+    @domain
+    Scenario: Filtering product by their type and size
+        When I filter them by double mug type and sticker size 7
```

Comment:
I think this scenario is quite unnatural, at first I have assumed that it should show only those products that are both double mugs and with sticker size 7.